### DISCUSSION TITLE
Should I start a new project on v3

### DISCUSSION THREAD

**Discussion by klacol** [dotnet/docfx#7741] [Q&A]
I am in the process of preparing a multiproject docs site with DocFx. I have the prototype up and running with v2; but there is a lot more work to to.

Should I change to v3 now, before making things too fixed?

**Comment by bitbonk** [comment]
DocFX v3 has been in development for *many* years and is already used in production for docs.microsoft.com for several years.
However it seems that the docfx team has abandoned the idea to invest in a v3 variant that is usable outside of docs.microsoft.com. 

So I'd suggest you stick with v2 or look into a docfx alternative like https://github.com/statiqdev

**Reply by KalleOlaviNiemitalo** [reply]
It's hard to even follow v3 development when pull requests refer to Microsoft private issues.

**Reply by DavidMarquezF** [reply]
At least now there still seems to be some activity. I was able to set up my companies docs without any problems and the new template theme they are working on looks quite similar to the Microsoft Docs one.

I don't know where I read it but I think they said we should have an update on how things will evolve quite soon

**Reply by bitbonk** [reply]
I am assuming that "activity" only exists to support docs.microsoft.com. I would not put my hopes too high: https://github.com/dotnet/docfx/issues/7663#issuecomment-1077353592

**Reply by KalleOlaviNiemitalo** [reply]
> they said we should have an update on how things will evolve quite soon

In February, <https://github.com/dotnet/docfx/issues/7050#issuecomment-1047519248> mentioned "something to share next quarter", i.e. 2022Q2.

**Reply by DavidMarquezF** [reply]
> I am assuming that "activity" only exists to support docs.microsoft.com. I would not put my hopes too high: [#7663 (comment)](https://github.com/dotnet/docfx/issues/7663#issuecomment-1077353592)

Well, there is the new template which I'm not sure why they would have any need for in docs.microsoft.com since they already have it working.

**Reply by FSharpCSharp** [reply]
Well, somehow I also find it all very strange. On the one hand, a more active development is now taking place again. But most of the commits come from some bots that update dependencies (very sinning, and often several times a day). On the other hand there are still branches for the open points like PDF generation speak. But these don't seem to have been completed yet, and the roadmap hasn't been updated for ages and three days. All in all, I personally find it very difficult to make a decision based on these facts.

**Reply by yufeih** [reply]
I know this is a confusing situation. While I'd like to get v3 shipped and have prepared a semi-working [prototype](https://github.com/docascode/template), we still need some time to finalize the plan this quarter.
